pkgs/development/beam-modules/elixir-ls/default.nix

```diff
@@ -21,7 +21,7 @@ mixRelease {
   mixFodDeps = fetchMixDeps {
     pname = "mix-deps-${pname}";
     inherit src version elixir;
-    hash = "sha256-q4VKtGxrRaAhtNIJFjNN7tF+HFgU/UX9sKq0BkOIiQI=";
+    hash = "sha256-yxcUljclKKVFbY6iUphnTUSqMPpsEiPcw4yUs6atU0c=";
   };
 
   # elixir-ls is an umbrella app
```

Comment:
line 40
as per stated in the https://github.com/elixir-lsp/elixir-ls#building-and-running
could you replace elixir_lst.release with elixir_ls.release2

the build method has been deprecated for a while and we have had to make the update for quite a while.
It shouldn't change anything else hopefully.

Reply:
I can, but the result is weird. After building elixir-ls with your suggested change, this is what happens when I try to run it:

```
$ result/bin/elixir-ls
Running /nix/store/dzmdzqvmszdgybix75ixqnndl9bv2c45-elixir-ls-0.19.0/bin/launch.sh
Preferred shell is bash, relaunching
Looking for ASDF install
ASDF not found
Looking for mise executable
which: no mise in (/nix/store/khndnv11g1rmzhzymm1s5dw7l2ld45bc-coreutils-9.4/bin:/nix/store/nbsz4b0kdh5a2b9h2cr9ngxskfqx575k-gnused-4.9/bin:/nix/store/mmsb0ivm355r4l3yjbpaiirkf673n66v-gnugrep-3.11/bin:/nix/store/bfb70iaascrajr78mfmr5kzspv0p7i6d-gawk-5.2.2/bin:/home/munksgaard/.config/guix/current/bin:/home/munksgaard/.local/bin:/home/munksgaard/.fly/bin:/home/munksgaard/.cabal/bin:/home/munksgaard/.cargo/bin:/home/munksgaard/.config/guix/current/bin:/home/munksgaard/.local/bin:/home/munksgaard/.fly/bin:/home/munksgaard/.cabal/bin:/home/munksgaard/.cargo/bin:/home/munksgaard/.config/guix/current/bin:/home/munksgaard/.local/bin:/home/munksgaard/.fly/bin:/home/munksgaard/.cabal/bin:/home/munksgaard/.cargo/bin:/run/wrappers/bin:/home/munksgaard/.nix-profile/bin:/nix/profile/bin:/home/munksgaard/.local/state/nix/profile/bin:/etc/profiles/per-user/munksgaard/bin:/nix/var/nix/profiles/default/bin:/run/current-system/sw/bin)
mise not found
Looking for rtx executable
which: no rtx in (/nix/store/khndnv11g1rmzhzymm1s5dw7l2ld45bc-coreutils-9.4/bin:/nix/store/nbsz4b0kdh5a2b9h2cr9ngxskfqx575k-gnused-4.9/bin:/nix/store/mmsb0ivm355r4l3yjbpaiirkf673n66v-gnugrep-3.11/bin:/nix/store/bfb70iaascrajr78mfmr5kzspv0p7i6d-gawk-5.2.2/bin:/home/munksgaard/.config/guix/current/bin:/home/munksgaard/.local/bin:/home/munksgaard/.fly/bin:/home/munksgaard/.cabal/bin:/home/munksgaard/.cargo/bin:/home/munksgaard/.config/guix/current/bin:/home/munksgaard/.local/bin:/home/munksgaard/.fly/bin:/home/munksgaard/.cabal/bin:/home/munksgaard/.cargo/bin:/home/munksgaard/.config/guix/current/bin:/home/munksgaard/.local/bin:/home/munksgaard/.fly/bin:/home/munksgaard/.cabal/bin:/home/munksgaard/.cargo/bin:/run/wrappers/bin:/home/munksgaard/.nix-profile/bin:/nix/profile/bin:/home/munksgaard/.local/state/nix/profile/bin:/etc/profiles/per-user/munksgaard/bin:/nix/var/nix/profiles/default/bin:/run/current-system/sw/bin)
rtx not found
Installing ElixirLS release v0.19.0
Running in /home/munksgaard/src/nixpkgs
* Getting elixir_ls (https://github.com/elixir-lsp/elixir-ls.git - v0.19.0)
* Getting elixir_sense (https://github.com/elixir-lsp/elixir_sense.git - e4c0282b8c2afa68652227992a223b1a9aab2c54)
* Getting erl2ex_vendored (https://github.com/elixir-lsp/erl2ex.git - 073ac6b9a44282e718b6050c7b27cedf9217a12a)
* Getting dialyxir_vendored (https://github.com/elixir-lsp/dialyxir.git - d50dcd7101c6ebd37b57b7ee4a7888d8cb634782)
* Getting jason_v (https://github.com/elixir-lsp/jason.git - c81537e2a5e1acacb915cf339fe400357e3c2aaa)
* Getting path_glob_vendored (https://github.com/elixir-lsp/path_glob.git - 965350dc41def7be4a70a23904195c733a2ecc84)
* Getting erlex_vendored (https://github.com/elixir-lsp/erlex.git - 82db0e82ee4896491bc26dec99f5d795f03ab9f4)
^C* Updating elixir_ls (https://github.com/elixir-lsp/elixir-ls.git - v0.19.0)
^C
BREAK: (a)bort (A)bort with dump (c)ontinue (p)roc info (i)nfo
       (l)oaded (v)ersion (k)ill (D)b-tables (d)istribution
```

(The breaks are my own)

It looks like it's trying to install itself from source?

Reply:
To be clear, I don't think this is desirable behavior from a nix package. For one, it might break reproducibility.